### PR TITLE
fix(TDI-39385): Show Salesforce GetUpdatedDeleted in wizzard

### PR DESCRIPTION
**Pull request by dmytro-sylaiev**
**What is the current behavior?** (You can also link to an open issue here)

Only SalesforceInput/Output/Bulk are shown when drag&drop salesforce module from wizzard

**What is the new behavior?**

Salesforce GetUpdated/GetDeleted added to shown list

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [x] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/1766/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/1766/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Adsylaiev%2FTDI-39385_add_salesforce_getUpdatedDeleted_toDragDropWizzard)

File: components/components-salesforce/components-salesforce-definition/src/main/java/org/talend/components/salesforce/tsalesforcegetdeleted/TSalesforceGetDeletedDefinition.java [hunk 1]
```diff
@@ -54,4 +56,9 @@ public RuntimeInfo getRuntimeInfo(ExecutionEngine engine, ComponentProperties pr
         return EnumSet.of(ConnectorTopology.OUTGOING);
     }
 
+    @Override
+    public Class<? extends ComponentProperties>[] getNestedCompatibleComponentPropertiesClass() {
+        return new Class[] { SalesforceConnectionProperties.class, SalesforceModuleProperties.class };
+    }
+
 }
```
Comment (olhava) [inline components/components-salesforce/components-salesforce-definition/src/main/java/org/talend/components/salesforce/tsalesforcegetdeleted/TSalesforceGetDeletedDefinition.java]: Please use same approach as in other components that are present in wizard module list (tSalesforceOutputBulkExec, tSalesforceInput, tSalesforceOutput)

File: components/components-salesforce/components-salesforce-definition/src/main/java/org/talend/components/salesforce/tsalesforcegetupdated/TSalesforceGetUpdatedDefinition.java [hunk 2]
```diff
@@ -54,4 +56,8 @@ public RuntimeInfo getRuntimeInfo(ExecutionEngine engine, ComponentProperties pr
         return EnumSet.of(ConnectorTopology.OUTGOING);
     }
 
+    @Override
+    public Class<? extends ComponentProperties>[] getNestedCompatibleComponentPropertiesClass() {
+        return new Class[] { SalesforceConnectionProperties.class, SalesforceModuleProperties.class };
+    }
```
Comment (olhava) [inline components/components-salesforce/components-salesforce-definition/src/main/java/org/talend/components/salesforce/tsalesforcegetupdated/TSalesforceGetUpdatedDefinition.java]: Same here (please look into (tSalesforceOutputBulkExec, tSalesforceInput, tSalesforceOutput))

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/1772/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/1772/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Adsylaiev%2FTDI-39385_add_salesforce_getUpdatedDeleted_toDragDropWizzard)

**Pull request merged**